__tests__/extension-requests/extension-requests.test.js

```diff
@@ -21,7 +21,7 @@ const {
 } = require('../../mock-data/users');
 const { usersStatus } = require('../../mock-data/users-status');
 const { taskDone, auditLogTasks } = require('../../mock-data/tasks/index');
-
+const baseUrl = 'http://localhost:8000/extension-requests';
```

Comment:
Do we also mention the port, host in UI tests? [not sure, just asking]

Reply:
Yes, this serves the required files for the UI